Lilu/Headers/kern_patcher.hpp

```diff
@@ -222,6 +222,69 @@ class KernelPatcher {
 
 		return (T)nullptr;
 	}
+    
+    /**
+     *  Solve request to resolve multiple symbols in one shot and simplify error handling
+     *
+     *  @seealso solveMultiple().
+     */
+    struct SolveRequest {
+        /**
+         *  The symbol to solve
+         */
+        const char *symbol {nullptr};
+        
+        /**
+         *  The symbol address on success, otherwise NULL.
+         */
+        mach_vm_address_t *address {nullptr};
+        
+        /**
+         *  Construct a solve request conveniently
+         */
+        template <typename T>
+        SolveRequest(const char *s, T &addr) :
+			symbol(s), address(reinterpret_cast<mach_vm_address_t*>(&addr)) { }
+    };
+	
+	/**
+	 *  Solve multiple functions with basic error handling
+	 *
+	 *  @param id        loaded kinfo id
+	 *  @param requests  an array of requests to solve
+	 *  @param num       requests array size
+	 *  @param start     start address range
+	 *  @param size      address range size
+	 *  @param crash     kernel panic on invalid non-zero address
+	 *
+	 *  @return false if at least one symbol cannot be solved.
```

Comment:
routeMultiple supports `force`, which is continue solving on failure. And it also runs `clearError` before returning unconditionally. Better to adopt it here as well.

Reply:
Done.